docs/samples/multimodelserving/triton/README.md

```diff
@@ -88,37 +76,66 @@ spec:
+{"level":"info","ts":1621717148.939014,"logger":"fallback-logger","caller":"agent/puller.go:136","msg":"Worker is started for cifar10"}
+{"level":"info","ts":1621717148.9393005,"logger":"fallback-logger","caller":"agent/puller.go:144","msg":"Downloading model from gs://kfserving-examples/models/torchscript/cifar10"}
+{"level":"info","ts":1621717148.939781,"logger":"fallback-logger","caller":"agent/downloader.go:48","msg":"Downloading gs://kfserving-examples/models/torchscript/cifar10 to model dir /mnt/models"}
+{"level":"info","ts":1621717149.4635677,"logger":"fallback-logger","caller":"agent/downloader.go:68","msg":"Creating successFile /mnt/models/cifar10/SUCCESS.71f376a9daa07a04ae1bd52cbe7f3a2c46ceb701350d9dffc73381df5a230923"}
+{"level":"info","ts":1621717149.6402793,"logger":"fallback-logger","caller":"agent/puller.go:161","msg":"Successfully loaded model cifar10"}
+{"level":"info","ts":1621717149.6404483,"logger":"fallback-logger","caller":"agent/puller.go:129","msg":"completion event for model cifar10, in flight ops 0"}
 ```
-Check the `Triton Inference Service` log you will see that the model is loaded into the memory
+
+Check the `Triton Inference Service` log you will see that the model is loaded into the memory.
 ```bash
-I1115 14:11:52.060489 1 model_repository_manager.cc:737] loading: cifar10:1
-I1115 14:11:52.061524 1 libtorch_backend.cc:217] Creating instance cifar10_0_0_cpu on CPU using model.pt
-I1115 14:11:52.690479 1 model_repository_manager.cc:925] successfully loaded 'cifar10' version 1
+I0522 20:59:09.469834 1 model_repository_manager.cc:737] loading: cifar10:1
+I0522 20:59:09.471278 1 libtorch_backend.cc:217] Creating instance cifar10_0_0_cpu on CPU using model.pt
+I0522 20:59:09.638318 1 model_repository_manager.cc:925] successfully loaded 'cifar10' version 1
 ```
 
-Now you can curl the model endpoint
+Check the `TrainedModel` CR status
+```bash
+kubectl get tm cifar10  
+NAME      URL                                                             READY   AGE
+cifar10   http://triton-mms.default.example.com/v2/models/cifar10/infer   True    3h45m
+
+# to show more detailed status
+kubectl get tm cifar10 -oyaml
+status:
+  address:
+    url: http://triton-mms.default.svc.cluster.local/v2/models/cifar10/infer
+  conditions:
+  - lastTransitionTime: "2021-05-22T20:56:12Z"
+    status: "True"
+    type: FrameworkSupported
+  - lastTransitionTime: "2021-05-22T20:56:12Z"
+    status: "True"
+    type: InferenceServiceReady
+  - lastTransitionTime: "2021-05-22T20:56:12Z"
+    status: "True"
+    type: IsMMSPredictor
+  - lastTransitionTime: "2021-05-22T20:58:56Z"
+    status: "True"
+    type: MemoryResourceAvailable
+  - lastTransitionTime: "2021-05-22T20:58:56Z"
+    status: "True"
+    type: Ready
+  url: http://triton-mms.default.example.com/v2/models/cifar10/infer
+```
+
+Now you can curl the model metadata endpoint
 ```bash
 MODEL_NAME=cifar10
 INPUT_PATH=@./input.json
```

Comment:
Does this input.json data file exist in this location?

Reply:
@shotarok I think here it is only calling the metadata endpoint so this input is not needed and I can remove it, I already included this input in the perf test job.